src/App.test.js

```diff
@@ -1,8 +1,22 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+/* eslint-disable testing-library/prefer-screen-queries */
```

Comment:
please follow our eslint recommnendations

You have to use global screen object to look for elements